spec/03-plugins/29-acme/05-redis_storage_spec.lua

```diff
@@ -256,7 +256,7 @@ describe("Plugin: acme (storage.redis)", function()
     end)
   end)
 
-  for _, strategy in helpers.each_strategy() do
+  for _, strategy in helpers.each_strategy({"postgres", "off"}) do
```

Comment:
it's not obvious to me why this change was needed

Reply:
There's flakyness triggered due to C*'s eventual consistency model. There's probably a more reliable way to ensure consistency of data but I figured we move away from Cassandra anyways and I didn't bother..